chapters/background.tex

```diff
@@ -1,11 +1,12 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-TensorFlow is a open source Machine Learning library developed by Google.
+The renaissance of machine learning in the 2010s outclassed this approach because it lacks the ability to detect emotions/smiles.
+Regarding soHappy, it led to TensorFlow becoming a more considerable option for emotion detection.
```

Comment:
Regarding the smile detection functionality of soHappy, Tensor flow  appears to be a convenient candidate.

Reply:
I'll be changing the whole thing here since it is linked to the first sentence